sunpy/visualization/animator/wcs.py

```diff
@@ -281,6 +282,9 @@ def plot_start_image_2d(self, ax):
                        'origin': 'lower'}
         imshow_args.update(self.imshow_kwargs)
 
+        if self.imshow_kwargs.get('vmax') == 'auto':
```

Comment:
Do we need to do this check twice?

Reply:
I guess this would work regardless of having to check it as we're passing ```None``` if not specified.